WordPress/src/main/java/org/wordpress/android/ui/accounts/login/nav/LoginNavHandler.java

```diff
@@ -0,0 +1,7 @@
+package org.wordpress.android.ui.accounts.login.nav;
+
+public interface LoginNavHandler {
+    void toast(String message);
```

Comment:
It's _kinda_ a temporary method but yes, there is a reason to have it in LoginNavHandler instead of using ToastUtils directly: Have a look at the `LoginNavController.java`. Notice that the module only imports from `java.util.Stack` and nothing else. That's by design, to steer clear of any Android or project dependencies. Its external functionality is provided via a clean interface (`LoginNavHandler`) alone. This makes the `LoginNavControler` testable on the JVM/Robolectric (by providing an implementation of LoginNavHandler in the tests) which is exactly the goal LoginNavController is introduced for: testability of the navigation logic.